### DISCUSSION TITLE
Documentation Guide

### DISCUSSION THREAD

**Discussion by Jamie98-ie** [moquette-io/moquette#856] [Q&A] [answered]
I can't find the documentation guide to set it up and use it. Can anyone help me?

**Comment by andsel** [comment]
Hi @Jamie98-ie starting from https://moquette-io.github.io/moquette/ -> `Documentation` you got to the documentation. It's little bit old, now Moquette is at version 0.18-SNAPSHOT and latest stable is 0.17, but that could be a good starting point.
If you find misses, please write here, I hope to provide more explanations.

**Reply by Jamie98-ie** [reply]
![Screenshot 2024-09-18 162017](https://github.com/user-attachments/assets/d14d84e0-ae59-4e7b-b834-b9e9201f2ee1)
there is no documentation upload on it

**Reply by andsel** [reply, marked as answer]
If you navigate the "Documentation" link should drive to https://moquette-io.github.io/moquette/documentation.html
